ConfigApp/Effects.cs

```diff
@@ -588,6 +589,7 @@ public enum EffectType
             {EffectType.EFFECT_PLAYER_AIMBOT, new EffectInfo("Aimbot", EffectCategory.PLAYER, "player_aimbot", true)},
             {EffectType.EFFECT_PEDS_SPAWN_BIKER, new EffectInfo("Spawn Biker", EffectCategory.PEDS, "peds_spawn_biker")},
             {EffectType.EFFECT_PEDS_SPAWN_SPACE_RANGER, new EffectInfo("Spawn Space Ranger", EffectCategory.PEDS, "peds_spawn_space_ranger")},
+            {EffectType.EFFECT_PEDS_ETER_SCREAMS, new EffectInfo("Eternal Screams", EffectCategory.PEDS, "peds_eter_screams")},
```

Comment:
id does not match the "correct" one  -> "peds_eternal_screams"
Please also check all other values you created, they're mostly "ETER_SCREAMS" or PedsEterScreams.cpp :)

Reply:
Ok, like this, right?

Reply:
Perfect 👌